### DISCUSSION TITLE
having issues with League.weeks()

### DISCUSSION THREAD

**Discussion by prestonlam53** [mattdodge/yahoofantasy#45] [Q&A]
hey everyone,

when I try to get league.weeks(), I get an error that says: 

  File "/Users/.../opt/anaconda3/envs/.../lib/python3.8/site-packages/yahoofantasy/resources/league.py", line 44, in weeks
    if not self.start_week or not self.end_week:
AttributeError: 'League' object has no attribute 'start_week'

Has anyone else faced this issue or knows why I'm getting an attribute error? I see in my league settings that there's a clear start and end date, and we go week 1 to week 21. Able to get other attributes about the league like standing data.

**Comment by mattdodge** [comment]
Hey @prestonlam53 this probably should be an issue, not a discussion, I'll move it over there. 

But I'm unable to reproduce this. Do you have a full code snippet you're running that I can try out on my end?

**Reply by prestonlam53** [reply]
### @mattdodge I actually solved this issue, it's the same as this thread. Thanks!

https://github.com/mattdodge/yahoofantasy/discussions/31
